.circleci/config.yml

```diff
@@ -0,0 +1,115 @@
+version: 2.1
+tagged_build_filters: &tagged_build_filters
+  branches:
+    ignore: /.*/
+  tags:
+    only: /v[0-9]+\.[0-9]+\.[0-9]+/
+test_build_filters: &test_build_filters
+  branches:
+    only: /.*/
+  tags:
+    ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.7.0
+    steps:
+      - checkout
+      - run: bundle install
+      - run: rubocop -c .rubocop.yml -a
+  build:
+    docker:
+      - image: circleci/ruby:2.7.0
+    steps:
+      - checkout
+      - run:
+          name: Check Tagged Push
+          command: |
+            PKG_VERSION=$(cat lib/logdna/version.rb | grep VERSION | cut -d'"' -f2)
```

Comment:
```suggestion
            PKG_VERSION=$(grep VERSION lib/version.rb | cut -d'"' -f2)
```

Reply:
@mindjiver, I'll commit if you change your suggestion to `PKG_VERSION=$(grep VERSION lib/logdna/version.rb | cut -d'"' -f2)`